releasenotes/notes/tracing:-ddtrace.constants.PID-changed-from-system.pid-to-process_id-197409ef9b68bd73.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
can we simplify the name of the release note to `system.pid-tag-change....yaml`?